### ISSUE TITLE
Batch normalization causes validation loss to fluctuate wildly

### ISSUE THREAD

**Issue by lminer** [keras-team/keras#10666]
I'm training a u-net model using batch normalization on all the convolution and deconvolution layers except for the last. For some reason the validation loss is going up and down by 50% or more from one epoch to the next. If I disable batch normalization, however, I don't see this phenomena. I'm having this problem with keras 2.1.4 - 2.2.0 and tensorflow 1.8 - 1.9. I don't see this issue in Chainer though training an identical network with identical data.

**Comment by lminer** [comment]
Seems to be related to this [issue](https://github.com/keras-team/keras/issues/7265). Turns out that the batch normalization momentum parameter in Chainer 0.9 rather than 0.99 in Keras. When I set momentum to 0.9, the issue is resolved.
